src/autoComplete/autoCompleteList.tsx

```diff
@@ -127,7 +127,7 @@ extends Component <AutoCompleteListProps, AutoCompleteListState> {
             loading={index === this.props.loadingItemIndex}
             select={this.props.onSelect}
             valid={index === this.props.loadingItemIndex && this.props.valid}
```

Comment:
I think you changed valid for status

Reply:
Hmm... It's the ItemChoice API, not the Button API. I'm not sure I want to impact too many components at once in this PR.

Reply:
Ok, I went into it: it's a property that adds a Button with the `CHECKED` status inside the `ItemChoice` (I think it's for the case with a loading ItemChoice), so I could probably rename it to `checked` or something, for consistency sake.